functions/__integrationtests__/handleFormSubmit.test.js

```diff
@@ -0,0 +1,71 @@
+jest.mock('cors');
+const testEnv = require('firebase-functions-test')(
+  {
+    projectId: 'dps-website-staging-0',
+  },
+  './dps-website-staging.firebase.key.json'
```

Comment:
We need to integrate this into the build process. :) 
Do you want to do it or should I create a PR for it?

Reply:
YoU :D